src/cascadia/TerminalApp/Command.cpp

```diff
@@ -29,13 +29,13 @@ namespace winrt::TerminalApp::implementation
 {
     Command::Command()
     {
-        _nestedCommandsView = winrt::single_threaded_vector<winrt::TerminalApp::Command>();
+        _subcommands = winrt::single_threaded_map<winrt::hstring, winrt::TerminalApp::Command>();

```

Comment:
Is it necessary to construct an empty map for every command, when the vast majority of them will not have subcommands?

Reply:
GAH okay, I'll do it.